orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -370,15 +369,16 @@ OrtValue CastToType(onnxruntime::ORTInvoker& invoker, const OrtValue& input, at:
   std::vector<OrtValue> output(1);
   NodeAttributes attrs(2);
   attrs["to"] = create_ort_attribute(
-    "to", GetONNXTensorProtoDataType(type), at::ScalarType::Long);
+      "to", GetONNXTensorProtoDataType(type), at::ScalarType::Long);
 
   auto status = invoker.Invoke("Cast", {
-    std::move(input),
-  }, output, &attrs);
+                                           std::move(input),
+                                       },
+                               output, &attrs);
 
   if (!status.IsOK())
     throw std::runtime_error(
-    "ORT return failure status:" + status.ErrorMessage());
+        "ORT return failure status:" + status.ErrorMessage());
```

Comment:
this formatting change irks me since it could fit on a line (and also because the conditional has to curly braces.  But I think we should replace these with CHECK_STATUS macro (but probably not in the reformat PR)

Reply:
Agree - Let's leave replacing for with CHECK_STATUS macro for future PR.